ApplicationRequirements.txt

```diff
@@ -3,13 +3,28 @@
 	"Password" Text Box
 	"Sign-in" Button
 		-> Takes to "Home" Screen if correct
+			-> Takes to the Driver variant of the "Home" Screen if account is registered to a driver
 		-> Redirects back to "Log-in" Screen if incorrect
 			-> Notifies user with red text
 	"Create New Account" Button
 		-> Takes to "Create New Account" Screen
 		
-"Home" Screen
-
+"Home" Screen - General
```

Comment:
I think general should be "User"

Reply:
Yeah, didn't know the best way to address users that aren't drivers because drivers are technically users too. I'm open to any suggestions on what they could be called.

Reply:
How can an account be a user account and a driver account? So they're both users with similar functionality. They can either create delivery requests or become pickup driver requests.

Reply:
The way I think about it, driver accounts are basically just extensions of the regular user accounts. You can do all the same stuff but also deliver stuff for people. In my opinion it’d be very frustrating to be a driver and have to log out and back in with a different account just to place a request.

Reply:
Yeah I agree with Sean. Also the initial registration for all accounts on our site will be the same. We need username, password, email, etc. A user can upgrade their account to a driver and we will ask them question about their car, etc. This way we dont confuse people with two registration forms. On the other hand a driver can access all the functionality a user can plus some extra things that are driver specific.